UNRELEASED.md

```diff
@@ -1,2 +1,7 @@
 # Unreleased
 
+**Fixed:**
+
+- bpk-component-calendar
+  - Fix edge cases with Week component
```

Comment:
One tiny thing - might be worth mentioning that this is specific to range-selection